base_m2m_custom_field/fields.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import fields
+
+
+class Many2manyCustom(fields.Many2many):
```

Comment:
what about `Many2manyNoTableCreate` (and change module name accordingly)?

Reply:
IMO this module could be improved with other slots than only `create_table`. That's also why I did set it to True as default...